pkg/fanal/analyzer/config/dockerfile/docker_test.go

```diff
@@ -1,62 +1,11 @@
 package dockerfile
 
 import (
-	"context"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-
-	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
-	"github.com/aquasecurity/trivy/pkg/fanal/types"
 )
 
-func Test_dockerConfigAnalyzer_Analyze(t *testing.T) {
```

Comment:
it might make sense to move this test (and for other config analyzers) to https://github.com/aquasecurity/trivy/pull/4090/files#diff-4c2c8a0bd35e3dc8f9517d5b759369cf430fdc682226b0bf85dbdcfb8386be3b or https://github.com/aquasecurity/trivy/pull/4090/files#diff-d1cc6bfaec51a49793b9c66ac339783f95b86c92cc15fdbb769001116e6c0e78R66

Reply:
Because we will not have tests to analyze the configuration after these changes.

Reply:
Misconfiguration scanning is well-tested here.
https://github.com/aquasecurity/trivy/blob/7bb20712fd1b1446aeb263422ccbe58cf11dcde6/pkg/fanal/artifact/local/fs_test.go

I've added a simple test for the config analyzer.
https://github.com/aquasecurity/trivy/pull/4090/commits/bace2209ec734219cfe980b7a617684a86f8550e

Reply:
cool!